omnibus/package-scripts/chef-server/postinst

```diff
@@ -52,6 +52,10 @@ if [ -e /etc/opscode/chef-server-running.json ]; then
   echo
   echo "For detailed upgrade instructions please see:"
   echo "https://docs.chef.io/server/upgrades/"
+else
+  echo -e "\033[1mThank you for installing Chef Infra Server!\033[0m"
```

Comment:
Should we include "The next step in install process is to run: `chef-server-ctl reconfigure`" here?

Reply:
We could probably say run this command and check out xyz.com for more information